firebase/auth.js

```diff
@@ -0,0 +1,15 @@
+var firebase = require('firebase');
```

Comment:
Tell me if firebase file must be save or delete.

Reply:
Yes you can delete firebase